i18n/src/test/scala/io/udash/i18n/TranslationKeyTest.scala

```diff
@@ -115,5 +116,21 @@ class TranslationKeyTest extends UdashSharedTest {
       val deserializedX = read[TranslationKey0](serializedX)
       getTranslatedString(deserializedX) should be(getTranslatedString(testKeyX((1, 2, "3", 4.5))))
     }
+
+    "have descriptive toString" in {
+      testKey0.toString should be("SimpleTranslationKey0(test0)")
+      testKey1.toString should be("TranslationKey1(test1)")
+      testKey2.toString should be("TranslationKey2(test2)")
+      testKey3.toString should be("TranslationKey3(test3)")
+      testKey4.toString should be("TranslationKey4(test4)")
+      testKey5.toString should be("TranslationKey5(test5)")
+      testKey6.toString should be("TranslationKey6(test6)")
+      testKey7.toString should be("TranslationKey7(test7)")
+      testKey8.toString should be("TranslationKey8(test8)")
+      testKey9.toString should be("TranslationKey9(test9)")
+      testKeyX.toString should be("TranslationKeyX(testX)")
+      testKeyU.toString should be("Untranslatable(testUntranslatable)")
+      testKeyR.toString should be("ReducedTranslationKey(test3,1,two,5)")
+    }
   }
```

Comment:
What happened to the equality / comparison tests? It's the initial reason for this changeset after all.

Reply:
Thought they are superfluous now, as case classes have a solid implementation of those. I can bring them back if you wish.

Reply:
I'd add them as a contract on what works on what doesn't, there are a couple corner cases with args and types hidden from the user.